CONTRIBUTING.md

```diff
@@ -1,6 +1,31 @@
-GraalVM welcomes contributors to the core platform and projects that extend that
-platform. There have been significant contributions from both industry
-and academia so far and we thank you for considering to contribute your changes!
+# Contributing to Mandrel
 
-- Learn [how to become a GraalVM contributor](https://www.graalvm.org/community/contributors/).
-- Subscribe and post to [graalvm-dev@oss.oracle.com](https://oss.oracle.com/mailman/listinfo/graalvm-dev) for questions related to working with the sources or extending the GraalVM ecosystem by creating new languages, tools, or embeddings.
+Mandrel is a downstream distribution of [GraalVM CE](www.github.com/oracle/graal).
+As a result contributions to Mandrel should be limited to:
+
+1. Backports from the upstream repository of GraalVM (e.g. bringing a feature/bug-fix from GraalVM CE 20.2 to Mandrel 20.1)
+2. Mandrel specific bug fixes (e.g. patches to make it work with upstream OpenJDK instead of Oracle's labs OpenJDK)
```

Comment:
Suggested phrasing: "... instead of Oracle Labs JDK"

Reply:
:+1: I will use `LabsJDK` as mentioned in https://github.com/graalvm/labs-openjdk-11/blob/master/README.md